proto/beacon/db/finalized_block_root_container.pb.go

```diff
@@ -5,11 +5,10 @@ package db
 
 import (
 	fmt "fmt"
+	proto "github.com/gogo/protobuf/proto"
```

Comment:
Can you run goimport?

Reply:
Ahh!   good to know.  Cause running` ./scripts/update-go-pbs.sh ` didn't fail   but i just noticed it ended with 
./scripts/update-go-pbs.sh: line 28: goimports: command not found
I'm investigating .

Reply:
goimports installed (golang x tools)